### PR TITLE
Simplify welcome screen

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

Screenshot 2020-05-06 at 14 54 44

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]

File: plugins/welcome/src/components/WelcomePage/WelcomePage.tsx [hunk 1]
```diff
@@ -44,24 +46,45 @@ const WelcomePage: FC<{}> = () => {
     <Page theme={pageTheme.home}>
       <Header
         title={`Welcome ${profile.givenName || 'to Backstage'}`}
-        subtitle="Some quick intro and links."
+        subtitle="Let's start build a better developer experience"
```
Comment (freben) [inline plugins/welcome/src/components/WelcomePage/WelcomePage.tsx:49]: `start to build` or `start building`

File: plugins/welcome/src/components/WelcomePage/WelcomePage.tsx [hunk 2]
```diff
@@ -33,9 +33,11 @@ import {
   pageTheme,
   ContentHeader,
   SupportButton,
+  WarningPanel,
 } from '@backstage/core';
-import ErrorButton from './ErrorButton';
-import ToggleFeatureFlagButton from './ToggleFeatureFlagButton';
+
+// import ErrorButton from './ErrorButton';
+// import ToggleFeatureFlagButton from './ToggleFeatureFlagButton';
```
Comment (freben) [inline plugins/welcome/src/components/WelcomePage/WelcomePage.tsx:40]: remove

File: plugins/welcome/src/components/WelcomePage/WelcomePage.tsx [hunk 3]
```diff
@@ -44,24 +46,45 @@ const WelcomePage: FC<{}> = () => {
...
+                    Milestones
+                  </Link>
+                  .
+                </>
+              }
+            />
+          </Grid>
           <Grid item xs={12} md={6}>
-            <InfoCard>
+            <InfoCard title="What now?">
```
Comment (freben) [inline plugins/welcome/src/components/WelcomePage/WelcomePage.tsx:80]: title case in english: `What Now?`

File: plugins/welcome/src/components/WelcomePage/WelcomePage.tsx [hunk 4]
```diff
@@ -85,7 +108,7 @@ const WelcomePage: FC<{}> = () => {
                 </ListItem>
               </List>
               <Typography variant="h6" gutterBottom>
-                Try It Out
+                Build your plugins
```
Comment (freben) [inline plugins/welcome/src/components/WelcomePage/WelcomePage.tsx:111]: title case in english: `Build Your Plugins`

**Comment by stefanalund** [comment]
@freben fixed, thanks!

**Review by freben — APPROVED** [review]

**Pull request merged**